postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -122,7 +122,13 @@ def __init__(self, check, config, shutdown_callback):
         # The value is loaded when connecting to the main database
         self._explain_function = config.statement_samples_config.get('explain_function', 'datadog.explain_statement')
         self._obfuscate_options = to_native_string(
-            json.dumps({'quantize_sql_tables': self._config.obfuscator_options.get('quantize_sql_tables', False)})
+            json.dumps(
+                {
+                    'quantize_sql': self._config.obfuscator_options.get(
```

Comment:
We could do this transformation/fallback-to-old-key once in `config.py` then here all we need to do is dump it to a string

Reply:
Updated: https://github.com/DataDog/integrations-core/pull/9884/commits/59c3e67d7f1d32419b7ca86956f94f1ad41974cf

Field name also changed for reasons explained here: https://github.com/DataDog/datadog-agent/pull/8860#discussion_r687479110